### DISCUSSION TITLE
[juliacall] how to broadcast a Julia function?

### DISCUSSION THREAD

**Discussion by Moelf** [cjdoris/PythonCall.jl#306] [General]
similar to https://github.com/cjdoris/PythonCall.jl/issues/285

**Comment by cjdoris** [comment]
Broadcasting has a special syntax in Julia which doesn't naturally translate to Python. You have a couple of options, either create a function to do the broadcasting:
```
f = jl.seval("x -> sqrt.(x)")
f(x)
```
or use the `broadcast` function directly:
```python
jl.broadcast(jl.sqrt, x)
```
